src/HospitalAPI/Emails/EmailService.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Net;
+using System.Net.Mail;
+using System.Threading.Tasks;
+using HospitalLibrary.Shared.Interfaces;
+
+namespace HospitalAPI.Emails
+{
+    public class EmailService : IEmailService
+    {
+        public async Task SendAppointmentEmail(string email)
+        {
+            MailMessage newMail = new MailMessage();
+            newMail.From = new MailAddress("levi.five.hospital@gmail.com", "Levi Five");
+            newMail.To.Add(email);
+            newMail.Subject = "Appointment assigned!";
+            newMail.Body = "<h2>Your appointment has been successfully added!<h2/><br/>" +
+                            "<h2><strong>Sent from LEVI-FIVE Hospital Service!</strong></h2>";
+            newMail.IsBodyHtml = true;
+            
+            SmtpClient client = new SmtpClient("smtp.gmail.com", 587);
+            client.EnableSsl = true;
+            NetworkCredential netCre = new NetworkCredential("levi.five.hospital@gmail.com", "fnaafrzlyeaxqupx");
+            client.Credentials = netCre;
+            client.Send(newMail);
```

Comment:
You can create client in constructor and send emails via client .send method. Another solution would be to make a method that returns client. Right now the logic is repeating itself.